snoop/data/management/commands/runworkers.py

```diff
@@ -72,7 +72,7 @@ def handle(self, *args, **options):
                 prefix = settings.TASK_PREFIX
             queues = options.get('func') or tasks.shaormerie
             system_queues = ['watchdog']
-            if options.get('synced'):
+            if bool_env(os.environ.get('SYNCED')):
```

Comment:
Let's call the env var `SYNC_FILES`. By the way, "sync" is preferrable to "synced", because it describes the action that snoop should take.

Reply:
Okay, I can see that.